packages/serverless/src/utils.ts

```diff
@@ -17,26 +17,12 @@ export function serverlessEventProcessor(event: Event): Event {
   return event;
 }
 
-/**
- * @returns Current active domain with a correct type.
- */
-export function getActiveDomain(): domain.Domain | null {
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any, @typescript-eslint/no-unsafe-member-access
-  return (domain as any).active as domain.Domain | null;
-}
-
 /**
  * @param fn function to run
  * @returns function which runs in the newly created domain or in the existing one
  */
 export function domainify<A extends unknown[], R>(fn: (...args: A) => R): (...args: A) => R | void {
```

Comment:
I would rather we directly use `runWithAsyncContext` everywhere instead of abstracting it, makes it easier to readers to understand what exactly is happening.